source/documentation/fuseki2/__index.md

```diff
@@ -66,7 +66,7 @@ preference to any older versions.
 | Filename | Description |
 |---------|-------------|
 |`apache-jena-fuseki-*VER*.zip` | Fuseki with UI download |
-|`[jena-fuseki-server](https://repo1.maven.org/maven2/org/apache/jena/jena-fuseki-server) | The Fuseki Main packaging |
+|`[jena-fuseki-server](https://repo1.maven.org/maven2/org/apache/jena/jena-fuseki-server)` | The Fuseki Main packaging |
```

Comment:
I normally use the backticks inside the link text, like [`jena-fuseki-server`](https://...), but either way is OK I think.

Reply:
Yes, it does (for Hugo, which we use) - fixed on the site.